### PR TITLE
[KS-54] OCR3 contract + config tooling

### PR DESCRIPTION
**Pull request by bolekk**
- Minimal contract serving OCR config.
- Script to generate OCR3-compatible config (heavily based on existing Functions scripts)

### CODE REVIEW

File: contracts/src/v0.8/keystone/OCR3Capability.sol [hunk 1]
```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
```
[the remark this answers is not in the document]
Comment (bolekk) [reply contracts/src/v0.8/keystone/OCR3Capability.sol:2]: will do!

[the remark this answers is not in the document]
Comment (archseer) [reply contracts/src/v0.8/keystone/OCR3Capability.sol:2]: feel free to assign me

[the remark this answers is not in the document]
Comment (bolekk) [reply contracts/src/v0.8/keystone/OCR3Capability.sol:2]: Yeah I don't think there's a team defined yet so let me assign several people for now.

File: CODEOWNERS [hunk 2]
```diff
@@ -66,6 +66,8 @@ core/scripts/gateway @smartcontractkit/functions
 /contracts/**/*llo-feeds* @smartcontrackit/mercury-team
```
Comment (bolekk) [inline CODEOWNERS:66]: Unrelated but cc @austinborn @samsondav

**Comment by bolekk** [comment]
Fixing lint and solhint...

File: CODEOWNERS [hunk 3]
```diff
@@ -66,6 +66,8 @@ core/scripts/gateway @smartcontractkit/functions
...
 /contracts/**/*vrf* @smartcontractkit/vrf-team
 /contracts/**/*l2ep* @smartcontractkit/integrations
+# TODO: replace with a team tag when ready
+/contracts/**/*functions* @archseer @bolekk @patrick-dowell
```
Comment (archseer) [inline CODEOWNERS:70]:
```suggestion
/contracts/**/*keystone* @archseer @bolekk @patrick-dowell
```

Comment (bolekk) [reply CODEOWNERS:70]: lol sorry

**Review by archseer — APPROVED** [review]

**Pull request merged**